src/shared/components/Tabs/Tabs.tsx

```diff
@@ -4,15 +4,20 @@ import React, { useEffect, useRef, useState } from 'react'
 import { CSSTransition } from 'react-transition-group'
 import { BackgroundGradient, Tab, TabsGroup, TabsWrapper, TAB_WIDTH } from './Tabs.styles'
 
+export type BadgeType = {
+  name: string
+  number: number
+}
 export type TabsProps = {
   tabs: string[]
   initialIndex?: number
   onSelectTab: (idx: number) => void
+  badges?: BadgeType[]
```

Comment:
Current implementation also allows us to have mismatch - we can update the tab name but forget to update the badge name and it will stop working. We should always try to avoid situations like that